style.css

```diff
@@ -510,8 +1063,567 @@ body {
+    margin-left: 50%;
+    margin-right: 0;
+  }
+
+  .bottom-section__image {
+    padding: 0;
+    margin-right: 40%;
+  }
+
+  .bottom-section__img {
+    margin: 137px 0;
+  }
+
+  .bottom-section__text-block {
+    -webkit-box-align: start;
+    -ms-flex-align: start;
+    align-items: flex-start;
+  }
+}
+
+@media (min-width: 1200px) {
+  .bottom-section__block {
+    margin-right: 7%;
+  }
+
+  .bottom-section__image {
+    margin-left: 3%;
+  }
+}
+
+@media (min-width: 992px) {
+  .bottom-section__title,
+  .bottom-section__subtitle,
+  .bottom-section__description {
+    margin-left: -80px;
+  }
+
+  .bottom-section__link {
+    position: absolute;
+    right: 8%;
+    bottom: 11%;
+    padding: 0;
+    width: 345px;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 13px;
+    line-height: normal;
+  }
+
+  .bottom-section__btn {
+    display: flex;
+    margin-top: -12.5%;
+    margin-left: 30%;
+    height: 43px;
+    text-align: center;
+    justify-content: center;
+  }
+
+  .bottom-section__arrow div {
+    bottom: 7px;
+    left: 240px;
+  }
+}
+
+@media (min-width: 1200px) {
+  .bottom-section__link {
+    right: 10%;
+    bottom: 21%;
+  }
+
+  .bottom-section__btn {
+    margin-top: -12.5%;
+    margin-left: 30%;
+  }
+
+  .bottom-section__arrow div {
+    bottom: 7px;
+    left: 240px;
+  }
+}
+
+@media (min-width: 770px) and (max-width: 992px) {
+  .bottom-section__title {
+    margin-left: -65px;
+  }
+
+  .bottom-section__subtitle {
+    margin-left: -40px;
+  }
+
+  .bottom-section__btn {
+    margin-left: 75px;
+  }
+}
+
+@media (min-width: 600px) and (max-width: 770px) {
+  .bottom-section__title {
+    margin-left: -475px;
+  }
+
+  .bottom-section__subtitle {
+    margin-left: -450px;
+  }
+
+  .bottom-section__description {
+    margin-left: -250px;
+  }
+}
+
+@media (min-width: 510px) and (max-width: 600px) {
+  .bottom-section__title {
+    margin-left: -270px;
+  }
+
+  .bottom-section__subtitle {
+    margin-left: -250px;
+  }
+
+  .bottom-section__description {
+    margin-left: -100px;
+  }
+}
+
+@media (min-width: 411px) and (max-width: 510px) {
+  .bottom-section__title {
+    margin-left: -40%;
+  }
+
+  .bottom-section__subtitle {
+    margin-right: 50%;
+  }
+
+  .bottom-section__description {
+    margin-left: 0px;
+  }
+}
+
+@media (min-width: 768px) and (max-width: 992px) {
+  .bottom-section__link {
+    display: inline-block;
+    margin-left: 600px;
+  }
+
+  .bottom-section__arrow {
+    display: block;
+    margin-top: 20px;
+    margin-left: 0;
+  }
+}
+
+@media (min-width: 500px) and (max-width: 768px) {
+  .bottom-section__link {
+    display: inline-block;
+    margin-left: 300px;
+  }
+
+  .bottom-section__arrow {
+    display: block;
+    margin-left: 0;
+    margin-top: 20px;
+  }
+}
+
+@media (max-width: 500px) {
+  .bottom-section__link {
+    display: inline-block;
+    margin-left: 188px;
+  }
+
+  .bottom-section__arrow {
+    display: block;
+    margin-top: 20px; 
```

Comment:
Codacy found an issue: [Expected newline after ";" in a multi-line declaration block (declaration-block-semicolon-newline-after)](https://app.codacy.com/gh/WebHeroSchool/locoalin/pullRequest?prid=5835770)